supervisor/utils/whoami.py

```diff
@@ -0,0 +1,58 @@
+"""Small wrapper for whoami API.
+
+https://github.com/home-assistant/whoami.home-assistant.io
+"""
+import asyncio
+from datetime import datetime
+import logging
+
+import aiohttp
+import attr
+
+from ..exceptions import WhoamiConnectivityError, WhoamiError, WhoamiSSLError
+from .dt import utc_from_timestamp
+
+_LOGGER: logging.Logger = logging.getLogger(__name__)
```

Comment:
I think that typing like this will be infered by MyPy. 
```suggestion
_LOGGER = logging.getLogger(__name__)
```